vspace/src/main/java/edu/asu/diging/vspace/core/model/impl/ChoiceBlock.java

```diff
@@ -0,0 +1,34 @@
+package edu.asu.diging.vspace.core.model.impl;
+
+import javax.persistence.Entity;
+import javax.persistence.OneToOne;
+
+import edu.asu.diging.vspace.core.model.IChoiceBlock;
+import edu.asu.diging.vspace.core.model.ISequence;
+
+@Entity
+public class ChoiceBlock extends ContentBlock implements IChoiceBlock {
+
+    @OneToOne(targetEntity = Sequence.class)
+    private ISequence sequence;
```

Comment:
I'm confused, your choice blocks should contain choices, not sequences, shouldn't they?